GiniVision/Classes/Core/CameraViewController.swift

```diff
@@ -614,15 +651,56 @@ extension CameraViewController {
     }
 }
 
+// MARK: - DocumentPickerCoordinatorDelegate
+
+extension CameraViewController: DocumentPickerCoordinatorDelegate {
+
+    func documentPicker(_ coordinator: DocumentPickerCoordinator,
+                        didPick documents: [GiniVisionDocument],
+                        from: DocumentPickerType,
+                        completion: DocumentPickerCompletion?) {
+        let loadingView = addValidationLoadingView()
+        self.validate(importedDocuments: documents) { validatedDocuments in
+            loadingView.removeFromSuperview()
+            let elementsWithError = validatedDocuments.filter { $0.1 != nil}
+            if let firstElement = elementsWithError.first,
```

Comment:
Why only the first error?

Reply:
Because this only applies for PDFs and QR codes, or when the multipage is not enabled. (The error is shown right away)